lib/spack/spack/build_systems/cmake.py

```diff
@@ -130,9 +130,9 @@ def _std_args(pkg):
             raise InstallError(msg)
 
         try:
-            build_type = pkg.spec.variants['build_type'].value
+            build_type = pkg.spec.variants['build_type'].value or ''
```

Comment:
I had no idea you could do this in Python!

Reply:
@scheibelp just showed it to me on a recent PR, it's great for keeping None checks clean!